ubyssey/urls.py

```diff
@@ -29,6 +32,9 @@
     # Elections
     url(r'^elections/$', theme.elections, name='elections'),
 
+    url(r'^calendar/$', theme.events_calendar, name='calendar'),
```

Comment:
I think I'd prefer to use `/events` for the calendar URL. Then we can keep the route structure consistent and do `events/(?P<event_id>[0-9]+)/$`  below